Automattic/wp-calypso
# Stats: Adding a gradient to Subscribers chart

Related to #75932

## Proposed Changes

* adding a gradient to Subscribers chart

## Testing Instructions



* Open the live branch for this PR
* Navigate to the traffic stats page at `/stats/day/:siteSlug`
* Add the query string `?flags=stats/subscribers-section` to enable the subscriber counts page
* Verify that the chart has a gradient under the main line
* Check if it looks fine when switching to `Month` or `Year` periods

| Before | After |
|---|---|
| ![SCR-20230426-mkvb](https://user-images.githubusercontent.com/112354940/234446512-64e451c9-a601-4849-8117-760089617462.png) | ![SCR-20230426-mkqv](https://user-images.githubusercontent.com/112354940/234446542-ce5cd0e8-91e3-4be3-9d07-6096070addc7.png) |



## Pre-merge Checklist
